tests/test_jwk_set.py

```diff
@@ -0,0 +1,205 @@
+import json
```

Comment:
You could merge this into `test_jwt`. Just add another view in the same fixture set.